src/Kconfig

```
@@ -37,6 +37,16 @@ config GOLIOTH_OTA_MAX_NUM_COMPONENTS
     help
         Maximum number of components in an OTA manifest.
 
+config GOLIOTH_FW_OBSERVATION_RETRY_MAX_DELAY_S
```

```suggestion
config GOLIOTH_OTA_OBSERVATION_RETRY_MAX_DELAY_S
```
I think `FW` can be ambiguous, as it really just means "firmware".